pymc/distributions/continuous.py

```diff
@@ -692,12 +692,12 @@ class TruncatedNormal(BoundedContinuous):
     @classmethod
     def dist(
         cls,
-        mu: Optional[Union[float, np.ndarray]] = None,
-        sigma: Optional[Union[float, np.ndarray]] = None,
-        tau: Optional[Union[float, np.ndarray]] = None,
-        sd: Optional[Union[float, np.ndarray]] = None,
-        lower: Optional[Union[float, np.ndarray]] = None,
-        upper: Optional[Union[float, np.ndarray]] = None,
+        mu: Optional[DIST_PARAMETER_TYPES] = None,
```

Comment:
Thanks for previous comment @ricardoV94. Hows it looking now?

One question, are dist parameter always optional? If so I can move the optional flag to the definition

Reply:
No they are not always optional. Most times they are required actually

Reply:
Hows it look now?

Reply:
Good!

Reply:
🎉  can we merge? Ill create an issue ticket for data umbrella sprint too have others add to the other parameters.

Thanks Ricardo